lang/well-formed.md

```diff
@@ -373,14 +373,14 @@ impl Function {
                 } else {
                     // A new block.
                     bb_live_at_entry.insert(block_name, live_locals);
-                    todo.push_back(block_name);
+                    todo.push(block_name);
```

Comment:
Or do you prefer `push_back` over `push`?
Generally, I tried to be as close to `Vec` as possible API-wise.

Reply:
`push` is fine.